### DISCUSSION TITLE
test bulb was to try out this idea

### DISCUSSION THREAD

**Discussion by rchiileea** [elax46/custom-brand-icons#309] [Ideas]
this is what I wanted that test bulb for, though about doing maybe picture/icon combo buttons in future

![CPT2301031458-688x783](https://user-images.githubusercontent.com/87599807/210382706-7d4f6c93-1a9a-4aec-8b26-7efacc2daef1.gif)

**Comment by elax46** [comment]
For the moment I have deleted it when you are ready to insert it will be done
